assets/javascript/script.js

```diff
@@ -95,4 +96,296 @@ function disappearMobileMenuContact() {
 disAppearMobileMenu.addEventListener('click', disappearMobileMenu);
 disAppearMobileMenuPortfolio.addEventListener('click', disappearMobileMenuPortfolio);
 disAppearMobileMenuAbout.addEventListener('click', disappearMobileMenuAbout);
-disAppearMobileMenuContact.addEventListener('click', disappearMobileMenuContact);
\ No newline at end of file
+disAppearMobileMenuContact.addEventListener('click', disappearMobileMenuContact);
+
+// Dynamic part of website
+
+const works = [
+  {
+    title: 'Tonic',
+    image: 'assets/images/tonic.svg',
+    imageD: 'assets/images/d-snapshot-3.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 0,
+    orderInside1: 'order-1',
+    orderInside2: 'order-2',
+  },
+  {
+    title: 'Multi-post stories',
+    image: 'assets/images/snapshoot-portfolio.svg',
+    imageD: 'assets/images/d-snapshot-2.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 1,
+    orderInside1: 'order-2',
+    orderInside2: 'order-1',
+  },
+  {
+    title: 'Tonic',
+    image: 'assets/images/snapshot-card-3.svg',
+    imageD: 'assets/images/d-snapshot-1.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 2,
+    orderInside1: 'order-1',
+    orderInside2: 'order-2',
+  },
+  {
+    title: 'Multi-post stories',
+    image: 'assets/images/snapshot-card-4.svg',
+    imageD: 'assets/images/snapshot-card-4.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 3,
+    orderInside1: 'order-2',
+    orderInside2: 'order-1',
+  },
+];
+
+const details = {
+  client: 'CANOPY',
+  role: 'Back End Dev',
+  counter: 'assets/images/counter.svg',
+  year: '2015',
+  description: 'This represents my expertise in Backend development.',
+  ppDscrptnD: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Non eum ex magni perspiciatis, praesentium quidem nemo officiis quas et quae esse voluptatum, minima asperiores perferendis repellat sint quaerat expedita quia!  Sunt assumenda mollitia perspiciatis accusantium illum sapiente soluta praesentium dolor ipsum debitis fugiat, doloribus blanditiis quo ut repellendus corrupti, nobis rem voluptas! Quod aut atque officiis hic esse cumque ad!',
+  ppDscrptnM: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
```

Comment:
- [ ] When the "See project" button is clicked on any project, the pop-up that appears has to be about that particular project. Currently, the same project and its details pop-up. I believe there is no need for this `details` object. Simply make use of your `works` array of objects and display the adequate pop-up modal 👍